packages/dara-core/js/shared/interactivity/data-variable.tsx

```diff
@@ -168,9 +168,10 @@ async function fetchDataVariableSchema(
     extras: RequestExtras,
     cacheKey?: string
 ): Promise<DataResponse['schema']> {
+    const queryString = new URLSearchParams({ cache_key: cacheKey }).toString();
     const response = await request(
-        `/api/core/data-variable/${uid}/schema`,
-        { body: JSON.stringify({ cache_key: cacheKey }), method: HTTP_METHOD.GET },
+        `/api/core/data-variable/${uid}/schema?${queryString}`,
```

Comment:
On the backend do we need to ensure the uriencoding is reverted? e.g. equivalent of decodeURIComponent. In case there's a mismatch this can cause it to miss the cache entry on the backend

Reply:
FastAPI automatically decodes all url-encoded query params